### ISSUE TITLE
Tab doesn't work normally in read-only present

### ISSUE THREAD

**Issue by Kazark** [rlister/org-present#23]
I would like to be able to slowly unfurl a slide as I go along, so I'd like to be able to tab the outline open and closed. It works somewhat, but the cursor jumps around and it doesn't entirely work, as if there is a bad interaction of two different bindings going on, or something.

**Comment by Kazark** [comment]
It works fine for a top-level header, but not when you move off it. As I press it on a top level header, I see echoed, `FOLDED`, then `CHILDREN`, then `SUBTREE`. But none of that gets echoed on a lower-level header.

**Comment by Kazark** [comment]
If I comment out the following code, this starts working again (though not everything else works well):

```
    ;; hide stars in headings
    (goto-char (point-min))
    (while (re-search-forward "^\\(*+\\)" nil t)
      (org-present-add-overlay (match-beginning 1) (match-end 1)))
```

**Comment by rlister** [comment]
Closed by #25 and #26
